docker-compose.yaml

```
@@ -12,39 +12,16 @@ services:
-      MYSQL_ROOT_PASSWORD: root
-      MYSQL_ALLOW_EMPTY_PASSWORD: 'true'
     volumes:
-      - ./traduora_mysql_data:/var/lib/mysql
+      - ./traduora_sqlite_data:/app/api/data
     networks:
       - overlay
 
 networks:
   overlay:
-    driver: bridge
+    driver: bridge
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)